src/foam/core/reflow/PropertyExprView.js

```diff
@@ -94,13 +94,16 @@ foam.CLASS({
             classNames: ['foam.lang.DateTime', 'foam.lang.Date'],
             label: 'Date Format',
             choices: [
-              [ null,                               '--'         ],
-              [ foam.mlang.expr.DateToHHExpr,       'HH'         ],
-              [ foam.mlang.expr.DateToHHMMExpr,     'HH:MM'      ],
-              [ foam.mlang.expr.DateToHHMMSSExpr,   'HH:MM:SS'   ],
-              [ foam.mlang.expr.DateToYYYYMMDDExpr, 'YYYY/MM/DD' ],
-              [ foam.mlang.expr.DateToYYYYMMExpr,   'YYYY/MM'    ],
-              [ foam.mlang.expr.DateToYYYYExpr,     'YYYY'       ]
+              [ null,                                  '--'           ],
+              [ foam.mlang.expr.DateToHHExpr,          'HH'           ],
+              [ foam.mlang.expr.DateToHHMMExpr,        'HH:MM'        ],
+              [ foam.mlang.expr.DateToHHMMSSExpr,      'HH:MM:SS'     ],
+              [ foam.mlang.expr.DateToYYYYMMDDExpr,    'YYYY/MM/DD'   ],
+              [ foam.mlang.expr.DateToYYYYMMExpr,      'YYYY/MM'      ],
+              [ foam.mlang.expr.DateToYYYYExpr,        'YYYY'         ],
+              [ foam.mlang.expr.DateToWeekExpr,        'Week (YYYY-W##)' ],
```

Comment:
Can we remove the #'s for these three?

Reply:
what should i replace them with? 

like YYYY-###, 

will be YYYY? its going to be confusing right?

Reply:
YYY-WWW
YYY-QQ
YYYY-DDD